tardis/io/atom_data/macro_atom_data.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+
+import pandas as pd
+
+
+@dataclass
+class MacroAtomData:
```

Comment:
Would be good to have docstrings that talk more about what the format of these dataframes should be